src/v/redpanda/application.cc

```diff
@@ -406,6 +406,14 @@ void application::setup_internal_metrics() {
           sm::description("Redpanda build information"),
           build_labels),
       });
+
+    _metrics.add_group(
+      "node:movements",
+      {sm::make_gauge(
+        "partition_movement_available_bandwidth",
+        [this] { return recovery_throttle.local().get_current_units(); },
+        sm::description(
+          "Bandwidth available for partition movement. bytes/sec"))});
```

Comment:
Please aggregate the shard label if you don't care about it:
`sm::make_gauge(..).aggregate({sm::shard_label})`.

Reply:
I think it makes sense to leave this metric per-shard because bandwidth quotas are per-shard. This way we can see if one of the shards has exhausted its quota which would be invisible in the aggregated metric.

Reply:
left without aggregation